src/DotNetWorker.Core/Context/FunctionContext.cs

```diff
@@ -59,5 +60,10 @@ public abstract class FunctionContext
         /// Gets a collection containing the features supported by this context.
         /// </summary>
         public abstract IInvocationFeatures Features { get; }
+
+        /// <summary>
+        /// Gets the <see cref="CancellationToken"/> that signals a function invocation is being cancelled.
+        /// </summary>
+        public virtual CancellationToken CancellationToken { get; }
```

Comment:
Should we provide a default value for this virtual property? 

`public virtual CancellationToken CancellationToken { get; } = CancellationToken.None;`